core/src/main/scala/org/dbpedia/extraction/ontology/OntologyDatatypes.scala

```diff
@@ -469,21 +469,21 @@ object OntologyDatatypes
 
         builder.addDimension("Temperature");
         builder.addUnit(new StandardUnitDatatype("kelvin", Set("K","kelvin",/*el*/ "Κ","κέλβιν")));
-        builder.addUnit(new FactorUnitDatatype("degreeCelsius", Set("°C","degree celsius","C","Celsius",/*el*/ "βαθμοί"), 1.0, 273.15));
+        builder.addUnit(new FactorUnitDatatype("degreeCelsius", Set("°C","degree celsius","C","Celsius",/*el*/ "βαθμοί", /*am*/ "ሴልሲየስ"), 1.0, 273.15));
         builder.addUnit(new FactorUnitDatatype("degreeFahrenheit", Set("°F","F","Fahrenheit","degree fahrenheit"), 5.0 / 9.0, 459.67));
         builder.addUnit(new FactorUnitDatatype("degreeRankine", Set("°R","R","degree rankine"), 5.0 / 9.0, 0));
```

Comment:
```suggestion
        builder.addUnit(new FactorUnitDatatype("degreesCelsius", Set("°C","degrees celsius","C","Celsius",/*el*/ "βαθμοί", /*am*/ "ሴልሲየስ"), 1.0, 273.15));
        builder.addUnit(new FactorUnitDatatype("degreesFahrenheit", Set("°F","F","Fahrenheit","degrees Fahrenheit"), 5.0 / 9.0, 459.67));
        builder.addUnit(new FactorUnitDatatype("degreesRankine", Set("°R","R","degrees Rankine"), 5.0 / 9.0, 0));
```

Reply:
agree, using degrees instead of degree is more accurate